### PR TITLE
Overview page refactor

### PR DESCRIPTION
**Pull request by jonathansberry**
Two updates for the overview page:
 - Fix the indicator order 
 - Use bootstrap grid in the standard box display.

### CODE REVIEW

**Comment by jonathansberry** [comment]
Thanks for spotting this.  Should have fixed it.

**Comment by jonathansberry** [comment]
You can now specify an indicator value as a percent or a number in the prep_details object.  See this pull request for a demo of how: https://github.com/meerkat-code/meerkat_jordan/pull/71

**Comment by Gulfa** [comment]
The latest commit fixed translation issues. Can you review as well Jonathan?

File: meerkat_frontend/src/js/technical/overview.js [hunk 1]
```diff
@@ -18,6 +18,12 @@ function html_box_builder(overviewObj, locID) {
     // But default to a DIV table format.
     var html_base = overviewObj.html_base || "<div class='row' id ='" + overviewObj.parentId + "'></div>";
 
+	html_base = html_base.replace(/<th>([\s\S]*?)<\/th>/g, 
+								  function(match, label) {
```
Comment (jonathansberry) [inline meerkat_frontend/src/js/technical/overview.js]: I've conceded that I was wrong about indentation when I started writing meerkat_frontend. I'm changing our javascript/html indentation to match our python indentation. I think you are still using tab characters perhaps?  Could you fix this as 4 spaces from now on?

Comment (jonathansberry) [reply meerkat_frontend/src/js/technical/overview.js]: Interesting solution though - I was assuming they would have to be translated with the html embedded in the string - like the frontpage strings. If it works, nice!

**Pull request merged**